MacrobenchmarkSample/functions/src/schema.ts

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
Technically this schema isn't macrobenchmark specific, except for the specific metric names.

suggest referring to this as general benchmark schema, but make a note in the `'metrics':{}` definition that it only supports macrobenchmark metrics currently.

Reply:
We can make this more flexible. This is not intended to be a general purpose schema definition.